libs/langchain/langchain/chat_models/mlflow.py

```diff
@@ -118,8 +118,10 @@ def _generate(
             "stop": stop or self.stop,
             "max_tokens": self.max_tokens,
             **self.extra_params,
+            **kwargs,
         }
-
+        if s := self.stop or stop:
```

Comment:
should stop be preferred to self.stop (ie `stop or self.stop`)?

Reply:
yes, updated!